src/Main/Main.js

```diff
@@ -0,0 +1,25 @@
+import './Main.scss'
+import burgerImage from './burgers.webp'
+import Navbar from '../Navbar/Navbar'
+import BannerBoxFoodDelivered from '../BannerBoxFoodDelivered/BannerBoxFoodDelivered'
+
+function Main() {
+  return (
+    <main className="Main">
+      <Navbar />
+      <div style={{backgroundImage: "url('" + burgerImage + "')"}} className="burger-image m-md-3">
+        <div className="row justify-content-center align-items-center h-100 m-3">
```

Comment:
as we understand it, rows must live inside a div/element with a container class https://stackoverflow.com/questions/19063976/bootstrap-grid-do-i-need-a-container